treeherder/etl/runnable_jobs.py

```diff
@@ -26,16 +26,15 @@ def _taskcluster_runnable_jobs(project, decision_task_id):
     validate = URLValidator()
     try:
         validate(tc_graph_url)
-        # `force_gzip_encoding` works around Taskcluster not setting `Content-Encoding: gzip`:
-        # https://bugzilla.mozilla.org/show_bug.cgi?id=1423215
-        tc_graph = fetch_json(tc_graph_url, force_gzip_decompression=True)
+        tc_graph = fetch_json(tc_graph_url)
     except ValidationError:
         logger.warning('Failed to validate %s', tc_graph_url)
         return []
     except requests.exceptions.HTTPError as e:
-        logger.info('HTTPError %s when getting taskgraph at %s',
+        logger.info('HTTPError %s when getting uncompressed taskgraph at %s',
                     e.response.status_code, tc_graph_url)
-        return []
+        logger.info('Attempting to fall back to the compressed taskgraph...')
```

Comment:
Could you add a TODO comment here to remove this handling once the `.json.gz` -> `.json` change has worked its way through the trains?

Reply:
Can do. 

How visible would this logger.info() be? Wonder if this should also log something into NewRelic or somewhere more visible so you can easily track how often the fallback is used?

Reply:
In production we only display warnings and above:
https://github.com/mozilla/treeherder/blob/56c1afe0a2106a7e269daee763f5b63299e14cc6/treeherder/config/settings.py#L207

So this could either be a warning or else it could manually be recorded in New Relic using either of:
https://docs.newrelic.com/docs/agents/python-agent/python-agent-api/record_exception
https://docs.newrelic.com/docs/agents/python-agent/python-agent-api/record_custom_event
(there are usages of both types in the repo)

Reply:
Happy to bikeshed on the message, but I think this would work and give all the information we'd want.